web/app/integration/wdio-sauce.conf.js

```diff
@@ -2,10 +2,19 @@ exports.config = {
   runner: 'local',
   user: process.env.SAUCE_USERNAME,
   key: process.env.SAUCE_ACCESS_KEY,
+  service: ['sauce'],
   sauceConnect: true,
   specs: [
-      './integration/specs/*.js'
+      './integration/specs/**/*.js'
   ],
+    suites: {
+    sidebar: [
+        './integration/specs/sidebar/*.js'
```

Comment:
I noticed that this commit includes 
```js
    sidebar: [
        './integration/specs/sidebar/*.js'
    ],
```

I don't see that directory in the integration directory. Was this added by mistake?

Reply:
Ah, this PR was actually meant to follow an older one(https://github.com/linkerd/linkerd2/pull/2698). I had made suites to allow for testing of various pages individually. I realized my error in submitting multiple PRs. I apologize for that. Will change this too.